### PR TITLE
Render OMIS order quote `accepted_by` only if defined

### PR DESCRIPTION
**Pull request by paulgain**
## Description of change
Display the `accepted_by` field for OMIS order quotes only when it contains a valid object, preventing potential issues with null.

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'

- [ ] Has the branch been rebased to main?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, Edge, Safari)

### CODE REVIEW

File: test/functional/cypress/specs/omis/quote-spec.js [hunk 1]
```diff
@@ -214,6 +214,13 @@ describe('Order quote', () => {
     })
   })
 
+  context('When the quote has been accepted, but accepted_by is null', () => {
+    it('should not render accepted-by-name', () => {
+      cy.visit(urls.omis.quote(quoteAccepted.id))
+      cy.get('[data-test="accepted-by-name"]').should('not.exist')
+    })
+  })
```
Comment (paulgain) [inline test/functional/cypress/specs/omis/quote-spec.js:222]: I'm unsure under what conditions this would be true: "When the quote has been accepted, but accepted_by is null". Could this be a BE issue?

Comment (baarkerlounger) [reply test/functional/cypress/specs/omis/quote-spec.js:222]: I'm also not sure if that's a valid case - likely an edge case at best. Possibly if the user the quote was accepted by has since been deleted this would happen?

I think rendering the rest of the component as you're doing here is the best approach though. If it's invalid data that at least makes it easier to surface and have removed.

**Review by baarkerlounger — APPROVED** [review]

**Review by dredmonds — APPROVED** [review]
